tests/unit/amo/pages/TestGuides.js

```diff
@@ -97,6 +110,11 @@ describe(__filename, () => {
       <a href="/en-US/android/collections/mozilla/password-managers/"
       class="Guides-section-explore-more-link">password manager</a> staff picks.</div>`,
     );
+
+    root.setProps({ addons });
```

Comment:
You are trying to test what happens when the add-ons are loaded, right? If so, this should become a new test.